src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './style.css';
+
+import Lights from './../Lights';
+import Climate from './../Climate';
+import Blinds from './../Blinds';
+import Energy from './../Energy';
+
+const Dashboard = ({ data }) => {
+  return (
+    <main className="dashboard">
+      <Lights lights={data.lights} />
+      <Climate
+        temperature={data.climate.temperature}
+        humidity={data.climate.humidity}
+      />
+      <Blinds blinds={data.blinds} />
```

Comment:
Bacha na to, že tady posíláš do komponenty Blinds prop jménem "blinds", ale v ní si pak vytahuješ "state" (která je undefined)
Proto ta tvoje komponenta pak nemá žádný počáteční stav (open ani close)
![image](https://user-images.githubusercontent.com/8430123/141322819-07544ff9-a854-4eaf-9c8f-38f563fdfb55.png)

Reply:
dakujem, zdalo sa mi to ze to vyzera nejako zvastne. tak uz viem preco